client/me/account/main.jsx

```diff
@@ -257,6 +255,40 @@ const Account = createReactClass( {
 		this.setState( { usernameAction: null } );
 	},
 
+	recordClickEvent( action ) {
+		this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
+	},
+
+	getClickHandler( action, callback ) {
+		return () => {
+			this.recordClickEvent( action );
+
+			if ( callback ) {
+				callback();
+			}
+		};
+	},
+
+	getFocusHandler( action ) {
+		return () => this.props.recordGoogleEvent( 'Me', 'Focused on ' + action );
+	},
+
+	getCheckboxHandler( checkboxName ) {
+		return event => {
+			const action = 'Clicked ' + checkboxName + ' checkbox';
+			const value = event.target.checked ? 1 : 0;
+
+			this.props.recordGoogleEvent( 'Me', action, 'checked', value );
+		};
+	},
+
+	getRadioHandler( radioName ) {
```

Comment:
This is only used to generate 1 handler. Could we just make a dedicated method to track that event?

Reply:
👍 makes sense - done in https://github.com/Automattic/wp-calypso/pull/22881/commits/e9bef140c2e2346bd1590bc16d4a3957e0cbefce